azure-pipelines.yml

```diff
@@ -119,11 +122,11 @@ stages:
           pathToPublish: '$(Build.ArtifactStagingDirectory)/packages'
           artifactName: 'NuGet Packages'
 
-      - task: ArtifactoryNuGet@1
+      - task: JFrogNuGet@1
         displayName: 'Publish SonarAnalyzer.CFG NuGet on repox'
         inputs:
           command: 'push'
-          artifactoryService: 'Repox artifactory'
+          artifactoryConnection: 'repox_qa_deployer_token'
```

Comment:
@malena-ebert-sonarsource  here I configured the artifactory connection, but for Maven deploy, I didn't change it...

I am not sure it uses the new connection?